plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/schema.py

```diff
@@ -182,7 +182,7 @@ def to_python_value(
         ctx: FlyteContext,
         lv: Literal,
         expected_python_type: Type[GreatExpectationsType],
-    ) -> GreatExpectationsType:
+    ) -> str:
```

Comment:
This is not a string either. It can be string, FlyteFile, or FlyteSchema.

Reply:
updated it, thanks.